spk/lidarr/src/service-setup.sh

```diff
@@ -0,0 +1,61 @@
+PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
```

Comment:
if ok with package dependency of mediainfo, add `/var/packages/mediainfo/target/bin`